data/tests/test_server_recordings_static.yaml

```diff
@@ -285,4 +285,59 @@ interactions:
     status:
       code: 200
       message: OK
+- request:
+    body: ra=0.0&dec=0.0&mjd_min=59227.66742310562&mjd_max=59592.66742310584&send_email=False
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      Authorization:
+      - Token testtoken
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '83'
+      Content-Type:
+      - application/x-www-form-urlencoded
+      User-Agent:
+      - python-requests/2.26.0
+    method: POST
+    uri: https://fallingstar-data.com/forcedphot/queue/?ra=0.0&dec=0.0&mjd_min=59227.66742310562&mjd_max=59592.66742310584&send_email=False
```

Comment:
When I hit this endpoint with the data specified here I get a 302 that redirects me to an authentication page.

So, is one call not enough to authorize and get the data?

Reply:
I don't think I understand :)